tests/test_ge_utils.py

```diff
@@ -282,52 +282,76 @@ def test_filter_properties_dict():
             properties=d0_begin,
             keep_fields=["c"],
             delete_fields=["a", "e"],
+            clean_falsy=True,
         )
-    d0_end: dict = filter_properties_dict(
-        properties=d0_begin,
-    )
+    d0_end: dict = filter_properties_dict(properties=d0_begin, clean_falsy=True)
     d0_end_expected = copy.deepcopy(d0_begin)
     d0_end_expected.pop("b")
     assert d0_end == d0_end_expected
 
     d1_begin: dict = copy.deepcopy(source_dict)
     d1_end: dict = filter_properties_dict(
         properties=d1_begin,
-        clean_empty=False,
+        clean_nulls=False,
     )
     d1_end_expected = copy.deepcopy(d1_begin)
     assert d1_end == d1_end_expected
 
     d2_begin: dict = copy.deepcopy(source_dict)
     d2_end: dict = filter_properties_dict(
         properties=d2_begin,
-        keep_fields=["b"],
+        clean_nulls=True,
+        clean_falsy=False,
     )
-    d2_end_expected = {"b": None}
+    d2_end_expected = copy.deepcopy(d2_begin)
+    d2_end_expected.pop("b")
     assert d2_end == d2_end_expected
 
     d3_begin: dict = copy.deepcopy(source_dict)
     d3_end: dict = filter_properties_dict(
         properties=d3_begin,
-        keep_fields=["a", "e"],
+        keep_fields=["b"],
+        clean_falsy=True,
     )
-    d3_end_expected = {"a": 0, "e": 9.8e1}
+    d3_end_expected = {"b": None}
     assert d3_end == d3_end_expected
 
     d4_begin: dict = copy.deepcopy(source_dict)
     d4_end: dict = filter_properties_dict(
         properties=d4_begin,
-        delete_fields=["a", "e"],
+        clean_falsy=True,
+        keep_falsy_numerics=False,
     )
-    d4_end_expected = {"c": "xyz_0", "d": 1}
+    d4_end_expected = copy.deepcopy(d4_begin)
+    d4_end_expected.pop("a")
+    d4_end_expected.pop("b")
     assert d4_end == d4_end_expected
 
     d5_begin: dict = copy.deepcopy(source_dict)
-    filter_properties_dict(
+    d5_end: dict = filter_properties_dict(
         properties=d5_begin,
+        keep_fields=["a", "e"],
+        clean_falsy=True,
+    )
+    d5_end_expected = {"a": 0, "e": 9.8e1}
+    assert d5_end == d5_end_expected
+
+    d6_begin: dict = copy.deepcopy(source_dict)
+    d6_end: dict = filter_properties_dict(
+        properties=d6_begin,
         delete_fields=["a", "e"],
+        clean_falsy=True,
+    )
+    d6_end_expected = {"c": "xyz_0", "d": 1}
```

Comment:
Just an optional suggestion - I think these tests would be easier to read if instead of "a", "b", "c", "d", "e" the keys of source_dict were descriptive, e.g. something like this:
```
source_dict: dict = {
        "0": 0,
        "None": None,
        "xyz_0": "xyz_0",
        "1": 1,
        "9.8e1": 9.8e1,
    }
```
or:
```
source_dict: dict = {
        "zero": 0,
        "none": None,
        "string": "xyz_0",
        "one": 1,
        "scientfic": 9.8e1,
    }
```

Reply:
This has been done.